lib/trellis/vagrant.rb

```diff
@@ -0,0 +1,64 @@
+# Set Ansible paths relative to Ansible directory
+ENV['ANSIBLE_CONFIG'] = ANSIBLE_PATH
+ENV['ANSIBLE_CALLBACK_PLUGINS'] = "~/.ansible/plugins/callback_plugins/:/usr/share/ansible_plugins/callback_plugins:#{File.join(ANSIBLE_PATH, 'lib/trellis/plugins/callback')}"
+ENV['ANSIBLE_FILTER_PLUGINS'] = "~/.ansible/plugins/filter_plugins/:/usr/share/ansible_plugins/filter_plugins:#{File.join(ANSIBLE_PATH, 'lib/trellis/plugins/filter')}"
+ENV['ANSIBLE_LIBRARY'] = "/usr/share/ansible:#{File.join(ANSIBLE_PATH, 'lib/trellis/modules')}"
+ENV['ANSIBLE_ROLES_PATH'] = File.join(ANSIBLE_PATH, 'vendor', 'roles')
+ENV['ANSIBLE_VARS_PLUGINS'] = "~/.ansible/plugins/vars_plugins/:/usr/share/ansible_plugins/vars_plugins:#{File.join(ANSIBLE_PATH, 'lib/trellis/plugins/vars')}"
+
+def fail_with_message(msg)
+  fail Vagrant::Errors::VagrantError.new, msg
+end
+
+def load_wordpress_sites
+  config_file = File.join(ANSIBLE_PATH, 'group_vars', 'development', 'wordpress_sites.yml')
+
+  if File.exists?(config_file)
+    wordpress_sites = YAML.load_file(config_file)['wordpress_sites']
+    fail_with_message "No sites found in #{config_file}." if wordpress_sites.to_h.empty?
+  else
+    fail_with_message "#{config_file} was not found. Please set `ANSIBLE_PATH` in your Vagrantfile."
+  end
+
+  wordpress_sites
+end
+
+def local_site_path(site)
+  File.expand_path(site['local_path'], ANSIBLE_PATH)
+end
+
+def multisite_subdomains?(wordpress_sites)
+  wordpress_sites.any? do |(_name, site)|
+    site['multisite'].fetch('enabled', false) && site['multisite'].fetch('subdomains', false)
+  end
+end
+
+def nfs_path(site_name)
+  "/vagrant-nfs-#{site_name}"
```

Comment:
This could be used for nfs path names in the `vagrant_synced_folders` loop:

```diff
- def nfs_path(site_name)
-   "/vagrant-nfs-#{site_name}"
+ def nfs_path(path)
+   "/vagrant-nfs-#{File.basename(path)}"
  end
```